spot_wrapper/wrapper.py

```diff
@@ -1311,16 +1311,29 @@ def velocity_cmd(
             v_rot: Angular velocity around the Z axis in radians
             cmd_duration: (optional) Time-to-live for the command in seconds.  Default is 125ms (assuming 10Hz command
                           rate).
+            body_height: Offset of the body relative to nominal stand height, in metres
 
         Returns:
             Tuple of bool success and a string message
         """
         end_time = time.time() + cmd_duration
-        response = self._robot_command(
-            RobotCommandBuilder.synchro_velocity_command(v_x=v_x, v_y=v_y, v_rot=v_rot, params=self._mobility_params),
-            end_time_secs=end_time,
-            timesync_endpoint=self._robot.time_sync.endpoint,
-        )
+
+        if body_height:
+            # If there is an offset body_height, use it to pose the robot
+            response = self._robot_command(
+                RobotCommandBuilder.synchro_velocity_command(v_x=v_x, v_y=v_y, v_rot=v_rot, body_height=body_height),
```

Comment:
Is it necessary to have an if condition here? Could the command just be this line regardless?

Reply:
I wasn't sure if there would be any case where it was important to pass in the other mobility params, so I included the second case. The stand function in this file has a similar condition structure. If the mobility params aren't used then I can definitely just use the first line.

Reply:
Ah missed that. A possible way you could condense without the if statement could be something like
```
mobility_params = None if stand_height else self._mobility_params
...
RobotCommandBuilder.synchro_velocity_command(
v_x=v_x, v_y=v_y, v_rot=v_rot,
stand_height=stand_height, params=mobility_params,
end_time_secs=end_time,
timesync_endpoint=self._robot.time_sync.endpoint)
```
but this is a minor nit